plugin.audio.mixcloud/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.audio.mixcloud"
        name="MixCloud"
-       version="2.1.3"
+       version="2.2.0"
        provider-name="jackyNIX">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
while you are on it can you please bump the xbmc.python version to 2.14.0 so it matches the gotham python api version?